indico/web/flask/session.py

```diff
@@ -74,14 +74,20 @@ def user(self):
                     flash(Markup(msg).format(merged_into_user.full_name), 'warning')
                 else:
                     flash(_(u'Your profile has been deleted.'), 'error')
+        if user and user.valid_login_num != self.get('_user_login_num'):
+            flash(_(u'Your session has been terminated.'), 'message')
```

Comment:
I'd use `warning` as the message type, and you need to clear before flashing - otherwise it's never shown.